Mage/Module.cs

```diff
@@ -67,28 +83,46 @@ public void SetupStaticData(Action<Exception> cb)
 		}
 
 
-		//
-		protected virtual string CommandPrefix
-		{
-			get { return null; }
-		}
+		// The module name as defined on the remote MAGE server
+		protected abstract string CommandPrefix { get; }
```

Comment:
The only reason why you would have no user commands is in the case where you only load static data - which is not part of any official MAGE SDK, since MAGE modules do not provide topics.